test/test_EKF_externalVision.cpp

```diff
@@ -204,3 +208,63 @@ TEST_F(EkfExternalVisionTest, visionAlignment)
 	EXPECT_TRUE(matrix::isEqual(externalVisionFrameOffset.canonical(),
 				    estimatedExternalVisionFrameOffset.canonical()));
 }
+
+TEST_F(EkfExternalVisionTest, velocityFrameBody)
+{
+	const Quatf quat_sim(Eulerf(0.0f, 0.0f, math::radians(90.0f)));
+	_sensor_simulator.simulateOrientation(quat_sim);
+	_sensor_simulator.runSeconds(2);
+
+	const Vector3f velVar_init = _ekf->getVelocityVariance();
+	EXPECT_NEAR(velVar_init(0), velVar_init(1), 0.0001);
+
+	_sensor_simulator._vio.setVelocityFrameToBody();
+
+	float vel_cov_data [9] = {2.0f, 0.0f, 0.0f,
+				  0.0f, 0.01f, 0.0f,
+				  0.0f, 0.0f, 0.01f};
+	const Matrix3f vel_cov_body(vel_cov_data);
+	const Vector3f vel_body(1.0f, 0.0f, 0.0f);
+	_sensor_simulator._vio.setVelocityCovariance(vel_cov_body);
+	_sensor_simulator._vio.setVelocity(vel_body);
+	_ekf_wrapper.enableExternalVisionVelocityFusion();
+	_sensor_simulator.startExternalVision();
+	_sensor_simulator.runSeconds(4);
+
+	const Vector3f velVar_new = _ekf->getVelocityVariance();
+	EXPECT_TRUE(velVar_new(1) > velVar_new(0));
```

Comment:
Rather than just saying "bigger", could you give some threshold/ratio you expect it to be bigger by, so this couldn't pass from random noise? Eg (not sure if these numbers are right, just a suggestion):
```suggestion
	EXPECT_TRUE(velVar_new(1) > 1.5f * velVar_new(0));
```

Reply:
done